code/game/atoms.dm

```diff
@@ -2135,12 +2135,18 @@
 						else if (rmb_text)
 							extra_context = rmb_text
 
-						// Ctrl-LMB and (in the future) Alt-LMB on another
+						// Ctrl-LMB
 						if (SCREENTIP_CONTEXT_CTRL_LMB in context)
 							if (extra_context != "")
 								extra_context += "<br>"
 							extra_context += "[SCREENTIP_CONTEXT_CTRL_LMB]: [context[SCREENTIP_CONTEXT_CTRL_LMB]]"
 
+						// Alt-LMB
+						if (SCREENTIP_CONTEXT_ALT_LMB in context)
+							if (extra_context != "")
```

Comment:
Can you do what I did for LMB/RMB? Ctrl-LMB and Alt-LMB should be on their own line, that's what the above comment meant.

It's all inlined because, y'know, super hot proc.